src/contract-metadata/L2StandardERC20ABI.json

```diff
@@ -0,0 +1,422 @@
+[
```

Comment:
One small nit from my side would be to minify this JSON to a single line and without spaces, since the current TS pipeline doesn't do this automatically for JSON files.

Reply:
I concur and update.